src/main.ts

```diff
@@ -0,0 +1,165 @@
+import { app, BrowserWindow, dialog } from "electron";
+import { readFileSync } from "fs";
+import * as path from "path";
+import * as process from "process";
+
+enum OS {
+  WINDOW = process.platform === "darwin" ? 0 : 1,
+  MAC = process.platform !== "darwin" ? 0 : 1,
+}
+
+const permittedExt: Array<string> = [".tt", ".oa", ".oc"];
+const mainnetOpenCert = "https://www.opencerts.io/";
+const mainnetTradeTrust = "https://www.tradetrust.io/";
+
+let macFilePath: string | null = null;
+
+// Return filePath
+function getFilePath(): string | null {
+  if ((process.env.NODE_ENV || "").trim() === "development") {
+    return path.join(app.getAppPath(), "assets/demo.tt");
+  }
+
+  if (OS.MAC) {
+    return macFilePath;
+  }
+
+  if (OS.WINDOW) {
+    // process.argv[0] is exe path
+    // process.argv[1] is open file path
+    if (process.argv.length >= 2) {
+      const windowFilePath = process.argv[1];
+      return windowFilePath !== "." ? windowFilePath : null;
+    }
+  }
+  return null;
+}
+
+function validateFile(filePath: string): string | null {
+  const extIndex = permittedExt.indexOf(path.extname(filePath));
+  switch (extIndex) {
+    case 0:
+    case 1:
+      return mainnetTradeTrust;
+    case 2:
+      return mainnetOpenCert;
+    default:
+      return null;
+  }
+}
+
+function createWindow(verifier: string): BrowserWindow | null {
+  // Create the browser window.
```

Comment:
```suggestion
function createBrowserWindow(verifier: string): BrowserWindow | null {
  // Create the browser window.
```

maybe then remove that comment?

Reply:
Done